packages/core/src/render3/instructions/control_flow.ts

```diff
@@ -43,6 +43,67 @@ import {
   removeLViewFromLContainer,
 } from '../view/container';
 
+export function ɵɵconditionalCreate(
```

Comment:
Similar to other instructions, we can add some docs and the `@codeGenApi` annotation:
```suggestion
/**
 * ...
 * @codeGenApi
 */
export function ɵɵconditionalCreate(
```